README.md

```diff
@@ -1 +1,30 @@
-# vcs-chat-recording
\ No newline at end of file
+# Daily chat + reaction recording with VCS
+
+This demo shows developers how they can embed chat and emoji reaction components into their recording of a Daily-powered video call.
+
+## In-call view
+![In-call view featuring chat and reactions](screenshot-app.png)
+
+## Recording view
+![Recording view featuring chat and reactions](screenshot-recording.png)
+
+
+## Running the demo locally
+
+You'll need a [free Daily account](https://dashboard.daily.co/u/signup) and [a Daily room](https://dashboard.daily.co/rooms/create) with cloud recording enabled on creation:
```

Comment:
I don't think recording is available through the free account 🤔

Reply:
(Just double checking what we can say on this one because I thought there might be some free credits for new accounts - will update when I confirm either way)

Reply:
Added a note here about cloud recording being pay-as-you-go